pyproject.toml

```diff
@@ -30,13 +30,13 @@ classifiers = [
     "Programming Language :: Python :: 3.10",
 ]
 dependencies = [
-    "zope.interface >= 4.4.2",
+    "zope.interface >= 5",
```

Comment:
with 4.4.2 was failing due to setuptools.

Now, maybe we should also pin a setuptools version.

I decided to ask for v5.

Reply:
Probably zope.interface should stop depending on setuptools instead

Reply:
it still does https://github.com/zopefoundation/zope.interface/blob/master/setup.py#L141

This was added in 2016 https://github.com/zopefoundation/zope.interface/pull/45 

It might no longer be valid